09-struct/conflation/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+)
+
+// START STRUCT OMIT
+type Circle struct {
+	X, Y, Radius int
+}
+// END STRUCT OMIT
+
+func main() { 
+// START BASIC OMIT
+	feet := 5.68
+	meters := 1.73
+	fmt.Println(feet + meters) // Is the result meaningful?
+
+	type Feet float64
+	type Meters float64
+	mile := Feet(5280)
+	km := Meters(1000)
+	// fmt.Println(mile + km) // type mismatch
```

Comment:
This brings up a good question for how we present code that doesn't compile. 

I did it differently in #5, maybe I should do this instead?